### ISSUE TITLE
Mail Rate

### ISSUE THREAD

**Issue by dev-brother** [soonick/poMMo#161]
I really need some help here. Currently my ISP limits the number of emails to 500 per hour. Currently my pommo is set to 10500 per hour. I just received a warning from my ISP about this.

The sliders do not work. How can I hard code this rate?

Thanks

**Comment by soonick** [comment]
Hello @swordsteel,

I thought I had fixed this issue in the past but it seems it broke again. I will take a look when I get some time. You should be able to change those values directly in the database. Inside your database you'll find a table named pommo_config which contains pommo configurations. You can search for the the row where config_name = throttle_MPS and change the value to 1 which would mean 1 mail per second but that is still 3600 per hour but that is the best we can do for now.

Regards,
Adrian

**Comment by dev-brother** [comment]
Hey Adrian

Okay, will have to do. I calculated that the MPS would have to set to 0.13 mps to hit out at 500 per hour. I understand that the current algorithm in poMMo doesn't work with fractions nor that it has a smaller timer increment less than 1 second to accomodate this. Maybe in future.

Thanks for the pointer, I will check it out.

I noticed under config_name there are four smtp entries. If they have valid smtp server names, will poMMo distribute its send between them? Is poMMo able to use multiple smtp servers for a send?

Later

Leon
